pkg/oc/cli/describe/projectstatus.go

```diff
@@ -355,16 +355,26 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 		} else if len(warningMarkers) > 1 {
 			warnings = fmt.Sprintf("%d warnings", len(warningMarkers))
 		}
+		if len(infoMarkers) > 0 {
+			infos = fmt.Sprintf("%d info", len(infoMarkers))
+		}
 
-		switch {
-		case !d.Suggest && len(errorMarkers) > 0 && len(warningMarkers) > 0:
-			fmt.Fprintf(out, "%s and %s identified, use '%[3]s status -v' to see details.\n", errors, warnings, d.CommandBaseName)
-
-		case !d.Suggest && len(errorMarkers) > 0 && errorSuggestions > 0:
-			fmt.Fprintf(out, "%s identified, use '%[2]s status -v' to see details.\n", errors, d.CommandBaseName)
+		markerStrings := []string{errors, warnings, infos}
+		markerString := ""
+		count := 0
+		for _, m := range markerStrings {
+			if len(m) > 0 {
+				if count > 0 {
+					markerString = fmt.Sprintf("%s, ", markerString)
+				}
+				markerString = fmt.Sprintf("%s%s", markerString, m)
+				count++
+			}
+		}
 
-		case !d.Suggest && len(warningMarkers) > 0:
-			fmt.Fprintf(out, "%s identified, use '%[2]s status -v' to see details.\n", warnings, d.CommandBaseName)
+		switch {
+		case !d.Suggest && ((len(errorMarkers) > 0 && errorSuggestions > 0) || len(warningMarkers) > 0 || len(infoMarkers) > 0):
+			fmt.Fprintf(out, "%s identified, use '%s status -v' to see details.\n", markerString, d.CommandBaseName)
```

Comment:
I don't see where the count for each of these is printed.

Reply:
Sure, I wasn't sure if it grammatically made more sense to have, eg, "2 infos identified" vs "2 info identified"

Reply:
Updated